### PR TITLE
Use parallelshell instead of our own watch

### PR DESCRIPTION
**Pull request by alicoding**

### CODE REVIEW

**Comment by alicoding** [comment]
@k88hudson R?

**Comment by alicoding** [comment]
tested on Windows and Mac and this is working perfectly \o/

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "build": "webpack --config webpack.config.js --progress --profile --colors",
     "watch-css": "node node_modules/autoless/bin/autoless --source-map templates/less public",
     "watch-js": "webpack --config webpack.config.js --progress --profile --colors --watch",
-    "watch": "node watch.js"
+    "watch": "parallelshell \"npm start\" \"npm run watch-css\" \"npm run watch-js\""
```
Comment (k88hudson) [inline package.json]: what do you thin about hving this be `npm start`, and making the current start `npm run serve`?

**Comment by alicoding** [comment]
@k88hudson done :D

File: package.json [hunk 2]
```diff
@@ -4,12 +4,12 @@
...
   "main": "index.js",
   "scripts": {
-    "start": "node web/index.js",
+    "serve": "node web/index.js",
     "test": "lab -t 100",
     "build": "webpack --config webpack.config.js --progress --profile --colors",
     "watch-css": "node node_modules/autoless/bin/autoless --source-map templates/less public",
     "watch-js": "webpack --config webpack.config.js --progress --profile --colors --watch",
-    "watch": "node watch.js"
+    "start": "parallelshell \"npm run serve\" \"npm run watch-css\" \"npm run watch-js\""
```
Comment (gvn) [inline package.json]: Does ctrl-c kill all these tasks?

Comment (alicoding) [inline package.json]: Confirmed that this is killed on both Windows and Mac :)

File: package.json [hunk 3]
```diff
@@ -13,7 +13,12 @@
     "jshint": "jshint  lib test web *.js",
     "jscs": "jscs lib test web *.js",
     "js-beautify": "js-beautify  lib test web *.js -r ",
-    "validate": "npm run jshint && npm run jscs"
+    "validate": "npm run jshint && npm run jscs",
+    "test": "lab -t 100",
+    "build": "webpack --config webpack.config.js --progress --profile --colors",
+    "watch-css": "node node_modules/autoless/bin/autoless --source-map templates/less public",
+    "watch-js": "webpack --config webpack.config.js --progress --profile --colors --watch",
```
Comment (k88hudson) [inline package.json]: did you add these in by accident?

Comment (alicoding) [inline package.json]: oops! Sorry I had them reorder by accident :P

**Comment by k88hudson** [comment]
r+

File: package.json (hunk 2 above)
Comment (gvn) [inline package.json]: :+1:

**Pull request merged**